src/app.rb

```diff
@@ -0,0 +1,105 @@
+require_relative './person'
+require_relative './book'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  private
+
+  def list_all_books
+    @books.each { |book| puts book }
+  end
+
+  def list_all_people
+    @people.each { |person| puts person }
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'That is not a valid input'
+      return
+    end
+
+    puts 'Person created successfully'
+  end
```

Comment:
- [ ] I think also you have an error here I commented the private also but it still there a problem with the arguments so kindly check them and try to solve the issues 🚀👌🏻💪🏻♻♻♻


![image](https://user-images.githubusercontent.com/58553711/200719770-f0401568-3fdf-4080-afba-7f0bb9a28b46.png)